src/ia32/lithium-ia32.cc

```diff
@@ -2980,6 +2970,15 @@ LInstruction* LChunkBuilder::DoBinarySIMDOperation(
         return DefineSameAsFirst(result);
       }
     }
+    case kFloat32x4ExtractLane:
+    case kFloat64x2ExtractLane:
+    case kInt32x4ExtractLane: {
+      LOperand* left = UseRegisterAtStart(instr->left());
+      LOperand* right = UseOrConstant(instr->right());
+      LBinarySIMDOperation* result =
+          new(zone()) LBinarySIMDOperation(left, right, instr->op());
+      return DefineAsRegister(result);
```

Comment:
As it might be deoptimized, please assign the environment. And you should combine the three cases into existing cases to avoid duplication, say`kFloat32x4Shuffle` case.